SharedProjects/Babylon2GLTF/GLTFExporter.Texture.cs

```diff
@@ -259,7 +259,7 @@ private GLTFTextureInfo ExportTexture(BabylonTexture babylonTexture, GLTF gltf,
                     gltfTextureInfo.scale = babylonTexture.level;
                 }
 
-                if (!(babylonTexture.uOffset == 0) || !(babylonTexture.vOffset == 0) || !(babylonTexture.uScale == 1) || !(babylonTexture.vScale == 1) || !(babylonTexture.wAng == 0))
+                if (!(babylonTexture.uOffset == 0) || !(babylonTexture.vOffset == 0) || !(babylonTexture.uScale == 1) || !(babylonTexture.vScale == -1) || !(babylonTexture.wAng == 0))
```

Comment:
Maybe it makes more sense to put this check inside the AddTextureTransformExtension after it inverts the y axis back?

Reply:
Yes, but in this case, as it has been designed, we loose the opportunity to warn the user when KHR_TextureTransform is unchecked. 
However, i do not understand why we have still this KHR_TextureTransform checkBox. This maybe has sense long time ago when the existing viewers did not support this extension.. but today, it's a non sense...

Reply:
You can just pass logger into the function.

Reply:
Wait, I thought logger is a variable, but maybe it's a namespace. Why do you loose the opportunity to warn the user?

Reply:
logger is a field of the GLTFExporter partial class, so it's available into the method
`private void AddTextureTransformExtension(..)`
Without logic change, this method is called only if use KHR_TextureTransform checkbox is checked.
The warning is raised only when unchecked.
```
if( Has Transform )
{
  if( enableKHRTextureTransform checked)
  {
     AddTextureTransformExtension()
  } 
 else 
 {
     warn the user
  }
}
```

We can for sure move the code, but this will change the actual logic and then we might want to change the method name to reflect the fact we MAY add the extension. 

I do not wrote this code, so either way are fine for me, i just choose the one with the less changes.

Reply:
This was just a comment. I'm okay if you merge it as is. It would be better if we didn't have the invert logic in multiple places.

Reply:
You're right, it's better to group logic. Here the new proposal.

Reply:
looks good